cmd.ExitStatus undefined

Hi,

(note I am not familiar with go). This is what I did:

```
go get github.com/Telmate/proxmox-api-go
cd ~/go/src/github.com/Telmate/proxmox-api-go/
go build -o proxmox-api-go
```

That seemed to work, next I did:

```
cd 
go get github.com/Telmate/terraform-provider-proxmox
```
And now I get the error:

```
# github.com/Telmate/terraform-provider-proxmox/proxmox
go/src/github.com/Telmate/terraform-provider-proxmox/proxmox/preprovision.go:165:8: cmd.ExitStatus undefined (type *remote.Cmd has no field or method ExitStatus, but does have remote.exitStatus)
go/src/github.com/Telmate/terraform-provider-proxmox/proxmox/preprovision.go:167:71: cmd.ExitStatus undefined (type *remote.Cmd has no field or method ExitStatus, but does have remote.exitStatus)
```


I'm using go version `go1.10.1 darwin/amd64`

--- a/proxmox/preprovision.go
+++ b/proxmox/preprovision.go
@@ -161,10 +161,8 @@ func runCommand(
 		return fmt.Errorf("Error executing command %q: %v", cmd.Command, err)
 	}
 
-	cmd.Wait()
-	if cmd.ExitStatus != 0 {
-		err = fmt.Errorf(
-			"Command %q exited with non-zero exit status: %d", cmd.Command, cmd.ExitStatus)
+	if err := cmd.Wait(); err != nil {
+		return err
 	}
 
 	// Wait for output to clean up

